src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -912,7 +916,10 @@ - (MTRBaseDevice *)deviceBeingCommissionedWithNodeID:(NSNumber *)nodeID error:(N
         return [[MTRBaseDevice alloc] initWithPASEDevice:deviceProxy controller:self];
     };
 
-    return [self syncRunOnWorkQueueWithReturnValue:block error:error];
+    MTRBaseDevice * device = [self syncRunOnWorkQueueWithReturnValue:block error:error];
+    MTR_LOG_DEFAULT("Getting device being commissioned with node ID 0x%016llX: %@ (error: %@)",
```

Comment:
I don't think what's effectively a getter should be logging this (or at least not at DEFAULT level). A client that's interested in this can easily log the return value of the call themselves.

Reply:
My issue is that I have clients who claim they are calling this function and I don't think they are, so I need log evidence for that....